source/contributing/how-to-get-help.md

```diff
@@ -0,0 +1,39 @@
+# How to get help
+
+If you need assistance with one of your contributions, there are a few places you
+can go for help.
+
+For better efficiency and chances of success, you should try to contact individuals or groups with more specific knowledge first:
+- Look at the [git log](https://www.git-scm.com/docs/git-log) for the files you need help
+  with, and take note of the email addresses of people who committed.
+- Check out if any teams are responsible for the relevant subsystem:
+  - On the [NixOS website](https://nixos.org/community/#governance-teams).
+  - In the [team list
+    file](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix).
+  - In the [Nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/.github/CODEOWNERS) or
+    [Nix](https://github.com/NixOS/nix/blob/master/.github/CODEOWNERS) `CODEOWNERS` files.
+- If your contribution is for a nixpkgs package, look for its maintainers in the
+  [`maintainers`](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers)
+  attribute.
+
+Once you have found the people you're looking for (either their email address, or their
+handles in [`maintainers-list.nix`][maintainers-list]), you can contact them via:
```

Comment:
```suggestion
Once you have found the people you're looking for, you can contact them via:
```

Since we go into details immediately, no need to repeat that specific information